gabbi/tests/test_inner_fixture.py

```diff
@@ -21,6 +21,8 @@
 import fixtures
 
 from gabbi import driver
+# TODO(cdent): this test_* needs to be imported bare or things do not work
```

Comment:
FWIW, I don't understand this note.

Reply:
Yeah, that's a copied note from the first time I was working through the pytest stuff. I should probably make it more clear. Basically pytest won't use the generator if the imported symbol is not in the module. The TODO-ness of it is bascially supposed to mean "ugh, figure out something to make this less gross."

All the pytest stuff is a big messy hack, it still awaits someone who knows what they are doing to come along, point and laugh, and then fix it.